install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -230,13 +231,51 @@ spec:
         securityContext:
           capabilities:
             add:
-            - NET_ADMIN
-            - SYS_MODULE
-          privileged: true
+              # Remove because we only need it to change the gid of the socket
+              # in the dev VM
+              # Note: check systemd options
+              # Note: check docker file to create user 'cilium'
+              # Check in security channel because the UIDs will be available in
+              # the host
+              - CHOWN
+              # put it back if something breaks
+              #- FOWNER
+              # put it back if something breaks - used for envoy
+              #- KILL
+              - NET_ADMIN
+              - NET_RAW
+              # Perf ring buffer
+              - IPC_LOCK
+              # Yes, for iptables. Consider removing once we are iptables-free
+              - SYS_MODULE
+              # We need it for now but might not need it for >= 5.11 specially
+              # for the '* override RLIMIT_NPROC resource limit'.
+              # In >= 5.8 we already have the BPF and PERMON
+              # Check if health endpoint still works because we need this:
+              # 'employ CLONE_* flags that create new namespaces with clone(2)
+              # and unshare(2) (but, since Linux 3.8, creating user namespaces
+              # does not require any capability);'
+              # Check if monitor breaks, we might need:
+              # 'access privileged perf event information;'
+              # Check if we still receive fsnotify events because of:
+              # 'call fanotify_init(2);'
+              - SYS_ADMIN
+              # Could be an alternative for the SYS_ADMIN for the RLIMIT_NPROC
+              - SYS_RESOURCE
+              # Re-add if endpoint create fails or if the log driver is using syslog
+              #- SYSLOG
+              # Requires kernel 5.8 and container runtime X
+              # if enabled user can remove SYS_ADMIN
+              #- PERFMON
+              #- BPF
         volumeMounts:
+        - mountPath: /host/proc/sys/net
```

Comment:
So with `privileged: true` the pod was getting `/proc/sys/net` and `/proc/sys/kernel` automatically? I'd say that it needs to be stated more explicitly in the commit message, as it looks like a significant change and not at all obvious. I would even consider putting it in a separate commit also, which can go before the change to `securityContext`.